src/accountConfigs/ornikar.ts

```diff
@@ -79,6 +79,7 @@ const config: Config<'dev' | 'design', 'ops' | 'frontends' | 'backends'> = {
       ChibiBlasphem: `christopher${process.env.ORNIKAR_EMAIL_DOMAIN}`,
       PSniezak: `paul.sniezak${process.env.ORNIKAR_EMAIL_DOMAIN}`,
       GaelFerrand: 'gael.ferrand@othrys.dev',
+      aenario: `aenario@gmail.com`,
```

Comment:
```suggestion
      aenario: `romain.foucault${process.env.ORNIKAR_EMAIL_DOMAIN}`,
```

c'est l'adresse email de slack